web/generate-geometries.js

```diff
@@ -593,6 +606,7 @@ const zoneDefinitions = [
   { zoneName: 'US-PJM', type: 'states', countryId: 'USA', states: [
     'US.PA', 'US.NJ', 'US.MD', 'US.DE', 'US.VA', 'US.WV', 'US.OH', 'US.KY']},
   // { zoneName: 'US-RI', countryId: 'USA', stateId: 'US.RI', type: 'state' },
+  { zoneName: 'US-SEC', type: 'county', counties: ['ALACHUA', 'BAKER', 'BRADFORD', 'CITRUS', 'CLAY', 'COLUMBIA', 'DESOTO', 'DIXIE', 'GADSDEN', 'GILCHRIST', 'GLADES', 'HAMILTON', 'HARDEE', 'HENDRY', 'HERNANDO', 'HIGHLANDS', 'JEFFERSON', 'LAFAYETTE', 'LAKE', 'LEON', 'LEVY', 'LIBERTY', 'MADISON', 'MANATEE', 'OKEECHOBEE', 'OSCEOLA', 'PASCO', 'PUTNAM', 'SARASOTA', 'SUMTER', 'SUWANNEE', 'TAYLOR', 'UNION', 'WAKULLA']},
```

Comment:
can you put this line after `US-SC` so it is ordered alphabetically?

Reply:
oops, yes, done.